tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/MalformedDockerConfigTest.java

```diff
@@ -64,26 +62,11 @@ public void testContainerWithHangingReadyCondition() throws Exception {
     public void testContainerReadyTimeout() throws Exception {
         Docker containerWithShortTimeout = new Docker.Builder("wildlfy", "jboss/wildfly:18.0.0.Final")
                 .setContainerReadyTimeout(2, TimeUnit.SECONDS)
-                .setContainerReadyCondition(() -> {
-                    try {
-                        URL url = new URL("http://" + "127.0.0.1:" + 9990 + "/health");
-                        HttpURLConnection connection = (HttpURLConnection) url.openConnection();
-                        connection.setRequestMethod("GET");
-
-                        return connection.getResponseMessage().contains("OK");
-                    } catch (Exception ex) {
-                        return false;
-                    }
-                })
-                .withPortMapping("8080:8080")
-                .withPortMapping("9990:9990")
-                .withCmdArg("/opt/jboss/wildfly/bin/standalone.sh")
-                .withCmdArg("-b=0.0.0.0")
-                .withCmdArg("-bmanagement=0.0.0.0")
+                .setContainerReadyCondition(() -> false) // never ready
                 .build();
 
         thrown.expect(DockerTimeoutException.class);
-        thrown.expectMessage(containsString("Container was not ready in timeout"));
+        thrown.expectMessage(containsString("Container was not ready in"));
 
         try {
             containerWithShortTimeout.start(); // will timeout after 1 sec
```

Comment:
> will timeout after 1 sec
Why? Isn't `timeout` set to 2 seconds?

Reply:
This is obsolete comment, I'll remove it.